source/recipe-individual/scripts/individualCustom.js

```diff
@@ -0,0 +1,295 @@
+            text-decoration: wavy;
+            color: black;
+            margin-left: 8%;
+            top: 0em;
+            margin-top: 9%;
+            padding-top: 0%;
+          }
+          h3{
+            font-family: 'Abril Fatface', cursive;
+            width: 8vw;
+            justify-self: left;
+          }
+          p {
+            font-family: 'Abril Fatface', cursive;
+          }
+
+          li {
+              font-family: 'Abril Fatface', cursive;
+          }
+
+          h2 {
+            font-family: 'Abril Fatface', cursive;
+          }
+        `;
+
+        styleElem.innerHTML = styles;
+        
+        const container = document.createElement('div');
+
+        const rightContainer = document.createElement('div');
+        rightContainer.setAttribute('class', 'application-type');
+        const thumbnail = document.createElement('img');
+        thumbnail.setAttribute("src",data.image);
+        const ingredientsContainer = document.createElement('div');
+        ingredientsContainer.setAttribute('class', 'essay-text');
+        const ingredientsHeadline = document.createElement('h2');
+        ingredientsHeadline.innerHTML = 'Ingredients';
+        const ingredientsText = document.createElement('ul');
+        for(var i = 0; i < data.extendedIngredients.length; i++) {
+            const single = document.createElement('li');
+            single.innerHTML = data.extendedIngredients[i].original;
+            ingredientsText.appendChild(single);
+        }
+        rightContainer.appendChild(thumbnail);
+        ingredientsContainer.appendChild(ingredientsHeadline);
+        ingredientsContainer.appendChild(ingredientsText);
+        rightContainer.appendChild(ingredientsContainer);
+
+        const topMiddleContainer = document.createElement('div');
+        topMiddleContainer.setAttribute('class', 'image-box');
+        const recipeTitle = document.createElement('h1');
+        recipeTitle.innerHTML = data.title;
+        const cookTime = document.createElement('h2');
+        cookTime.innerHTML = data.readyInMinutes + "min";
+        // const description = document.createElement('p');
+        // description.innerHTML = data.
+        const ratingBox = document.createElement('div');
+        ratingBox.setAttribute('class', 'ratings-box');
+        const starsDiv = document.createElement('div');
+        const stars = document.createElement('img');
+        stars.setAttribute('src', './images/5-star.svg');
+        stars.setAttribute('class', 'stars');
+        starsDiv.appendChild(stars);
+        const ratingsDiv = document.createElement('div');
+        const ratings = document.createElement('p');
+        ratings.innerHTML = data.aggregateLikes + " Likes";
+        ratingsDiv.appendChild(ratings);
+        const favorite = document.createElement('a');
+        favorite.setAttribute('class', 'favorite');
+        favorite.setAttribute('href', '');
+        const favImage = document.createElement('img');
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/Big-Bytez/cse110-fa21-group19/pullRequest?prid=8445170)
```suggestion
        const favImage = document.createElement("img");
```